server/pulp/server/async/scheduler.py

```diff
@@ -425,4 +473,5 @@ def apply_async(self, entry, publisher=None, **kwargs):
             has_failure = bool(entry._scheduled_call.consecutive_failures)
             self._failure_watcher.add(result.id, entry.name, has_failure)
             _logger.debug(_('watching task %s') % {'id': result.id})
+            _logger.info(_('watching task %s') % {'id': result.id})
```

Comment:
On 06/19/2015 03:40 PM, bmbouter wrote:
> This should be logged at either |debug| or |info| but not both. I'll
> suggest doing it at |info| so keep this line and delete the one above it.

IMO this one makes sense to stay at debug. I don't think it will be
meaningful to users, and it adds noise to the log for most users.

-- 
Randy Barlow